backend/src/main/scala/cromwell/backend/package.scala

```diff
@@ -35,6 +35,9 @@ package object backend {
                                   key: BackendJobDescriptorKey,
                                   inputs: Map[LocallyQualifiedName, WdlValue]) {
     val call = key.call
+
+    /** The abort functionality potentially becomes available after the job descriptor is created */
+    var abortFunction: Option[() => Unit] = None
```

Comment:
I don't know guys, this is getting kinda complicated.. I need to override `receive` on two actors and it's still not clear to me how to actually get the Run ID from the metadata service... it might take me a while to figure that out :-/  #notsmartenoughforcromwell